02-research_team.Rmd

```diff
@@ -0,0 +1,36 @@
+# Research Team
+
+This project is coordinated by the Genomics Data Science Community Network (GDSCN). You can read more about the GDSCN and its mission at the network [website](https://www.gdscn.org/home).
+
+## Soil sampling 
+
+Soil sampling for this project was done by both faculty and student volunteers from schools that aren't traditional R1 research institutions. Many of the faculty are also members of the GDSCN.
```

Comment:
Could add some reference to the R1, etc tier system. I think there's a ref in the GDSCN Genome Research paper that could help!